client_tools.py

```diff
@@ -49,9 +49,8 @@ def parse_command_line():
 
     parser.add_argument('-m', '--md5', nargs="?", type=str,  help='md5 batch to remove from database', default=None)
 
-    parser.add_argument('-i', '--img_purge', nargs="?", type=bool, help='Setting to only purge image attachments', default=False)
-
-    parser.add_argument('-f', '--full_import', nargs="?", type=bool, help='Set to True if doing an import that imports both data and images',
+    parser.add_argument('-f', '--full_import', nargs="?", type=bool, help='Set to True if doing an '
+                                                                          'import that imports both data and images',
```

Comment:
removed img purge argparser arg, instead of purging just parts of upload batches, just puring the entire batch is simpler and less prone to error.